ios/RNZoomBridge.m

```diff
@@ -208,7 +208,7 @@ - (void)onMeetingReturn:(MobileRTCMeetError)errorCode internalError:(NSInteger)i
     return;
   }
 
-  if (errorCode != MobileRTCMeetError_Success) {
+  if (errorCode != MobileRTCMeetError_Success || errorCode != 0) {
```

Comment:
I believe 0 == MobileRTCMeetError_Success

https://marketplace.zoom.us/docs/sdk/native-sdks/iOS/resource/error-codes

Can we also add `MobileRTCMeetError_MeetingOver`. There is no equivalent to MEETING_ERROR_EXIT_WHEN_WAITING_HOST_START

So I think we should change this to:

` if ((errorCode != MobileRTCMeetError_Success) && (errorCode != MobileRTCMeetError_MeetingOver)) {`